WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -208,6 +204,14 @@ - (void)willTransitionToTraitCollection:(UITraitCollection *)newCollection withT
 
 #pragma mark - Data Model setup
 
+- (NSString *)adminRowTitle
+{
+    if (self.blog.isHostedAtWPcom) {
+        return NSLocalizedString(@"Dashboard", @"Action title. Noun. Opens the user's WordPress.com dashboard in an external browser.");
```

Comment:
Hmm.. good point.  @mattmiklic do you have any suggestions for naming?  I think we should avoid anything with "Calypso" but, yah, "Dashboard" maybe isn't the best fit.